cengage/react-magma
# feat: Pagination variant

Issue: # [991](https://github.com/cengage/react-magma/issues/991)

## What I did
- New pagination variant adapting pagination into a dropdown selector.

## Screenshots (if appropriate):
![hooray](https://user-images.githubusercontent.com/77400920/227005877-40286e31-fc2e-4aa4-8de1-e6efa17eac0c.png)

## Checklist 

## How to test

- Make sure the type `PaginationType.simple` works as a dropdown with two navigation buttons.
- Ensure the doc site has the appropriate changes.
